### ISSUE TITLE
Mulldav exit node not appearing in Windows

### ISSUE THREAD

**Issue by FloatingLizard** [tailscale/tailscale#13543]
### What is the issue?

Exactly like the issue #13472 
option not present in the Aplication Windows and of course the device enabled and configured

### Steps to reproduce

Added the device several times

### Are there any recent changes that introduced the issue?

No

### OS

Windows

### OS version

Windows 10

### Tailscale version

1.74.0

**Comment by FloatingLizard** [comment]
I have deleted all the devices and added back on and now none of them are working.

**Comment by FloatingLizard** [comment]
The application from IOS and Android and Linux are working again.  But Windows application simply do not registers.

**Comment by kelivel** [comment]
Can you paste a bugreport generate from the affected Windows machine?

**Comment by FloatingLizard** [comment]
I don't suppose there is a way to generate a log from within the app/or is there?

On September 23, 2024 6:51:51 PM GMT+02:00, Keli ***@***.***> wrote:
>Can you paste a bugreport generate from the affected Windows machine?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/tailscale/tailscale/issues/13543#issuecomment-2368836052
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
Saves Yourselves

**Comment by FloatingLizard** [comment]
I archived all the logs

On September 23, 2024 6:51:51 PM GMT+02:00, Keli ***@***.***> wrote:
>Can you paste a bugreport generate from the affected Windows machine?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/tailscale/tailscale/issues/13543#issuecomment-2368836052
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
Saves Yourselves

**Comment by kelivel** [comment]
On Windows:

- Hold the Ctrl key and right-click the Tailscale icon in the menu bar.
- Select Debug and then select Bug report.
- Select Copy report ID to copy the bug report.

https://tailscale.com/kb/1227/bug-report?tab=windows

**Comment by FloatingLizard** [comment]
BUG-31414d82cb9f13f6bbce21f54aecda6712a364e2bebbc50fdaaecd7d5ce34ade-20240923172922Z-c07278df3550cd42

On September 23, 2024 7:26:30 PM GMT+02:00, Keli ***@***.***> wrote:
>On Windows:
>
>- Hold the Ctrl key and right-click the Tailscale icon in the menu bar.
>- Select Debug and then select Bug report.
>- Select Copy report ID to copy the bug report.
>
>https://tailscale.com/kb/1227/bug-report?tab=windows
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/tailscale/tailscale/issues/13543#issuecomment-2368914697
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
Saves Yourselves

**Comment by FloatingLizard** [comment]
Any feedback?

On September 23, 2024 7:26:30 PM GMT+02:00, Keli ***@***.***> wrote:
>On Windows:
>
>- Hold the Ctrl key and right-click the Tailscale icon in the menu bar.
>- Select Debug and then select Bug report.
>- Select Copy report ID to copy the bug report.
>
>https://tailscale.com/kb/1227/bug-report?tab=windows
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/tailscale/tailscale/issues/13543#issuecomment-2368914697
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
Saves Yourselves

**Comment by kelivel** [comment]
Did you make any recent changes to the [Tailnet Lock](https://tailscale.com/kb/1226/tailnet-lock) status on your tailnet? The device in your bugreport is waiting for a signature before it can connect to other devices, including Mullvad exit node endpoints:

`user bugreport health: this node is locked out; it will not have connectivity until it is signed. For more info, see https://tailscale.com/s/locked-out`

If you still have Tailnet Lock enabled, the script at https://github.com/tailscale-support/mullvad-script will sign all of the Mullvad nodes so your device can connect to them. Otherwise, if you've actually _disabled_ Tailnet Lock, there's a known bug where devices may not sync the new status (and continue to be locked after the feature is disabled), and you'll need to remove and re-add the device to your tailnet to get it into the correct state.

**Comment by FloatingLizard** [comment]
How can I solve the issue?On 23 Sep 2024, at 22:47, Keli ***@***.***> wrote:﻿
Did you make any recent changes to the Tailnet Lock status on your tailnet? The device in your bugreport is waiting for a signature before it can connect to other devices, including Mullvad exit node endpoints:
user bugreport health: this node is locked out; it will not have connectivity until it is signed. For more info, see https://tailscale.com/s/locked-out
If you still have Tailnet Lock enabled, the script at https://github.com/tailscale-support/mullvad-script will sign all of the Mullvad nodes so your device can connect to them. Otherwise, if you've actually disabled Tailnet Lock, there's a known bug where devices may not sync the new status (and continue to be locked after the feature is disabled), and you'll need to remove and re-add the device to your tailnet to get it into the correct state.

**Comment by FloatingLizard** [comment]
Thank you for your help!  It’s resolved!On 23 Sep 2024, at 22:47, Keli ***@***.***> wrote:﻿
Did you make any recent changes to the Tailnet Lock status on your tailnet? The device in your bugreport is waiting for a signature before it can connect to other devices, including Mullvad exit node endpoints:
user bugreport health: this node is locked out; it will not have connectivity until it is signed. For more info, see https://tailscale.com/s/locked-out
If you still have Tailnet Lock enabled, the script at https://github.com/tailscale-support/mullvad-script will sign all of the Mullvad nodes so your device can connect to them. Otherwise, if you've actually disabled Tailnet Lock, there's a known bug where devices may not sync the new status (and continue to be locked after the feature is disabled), and you'll need to remove and re-add the device to your tailnet to get it into the correct state.

**Comment by FloatingLizard** [comment]
Did you make any recent changes to the [Tailnet Lock](https://tailscale.com/kb/1226/tailnet-lock) status on your tailnet? The device in your bugreport is waiting for a signature before it can connect to other devices, including Mullvad exit node endpoints:

 " I did activate the tailnet lock with the previous Tailnet account in which I deleted and I then created a new account and the Application was still pointing to the tailnet lock that was activated with the previous account and that was the issue.
  
  I deleted the application + 
C:\ProgramData\Tailscale
C:\Users\%USERNAME%\AppData\Local\Tailscale
C:\Windows\System32\config\systemprofile\AppData\Local\Tailscale

And that resolved the issue.

Again thank you for your help.
